crypto3/libs/blueprint/include/nil/blueprint/bbf/generic.hpp

```diff
@@ -325,7 +325,7 @@ namespace nil {
 
                 void allocate(TYPE &C, size_t col, size_t row, column_type t) {
                     if (is_allocated(col, row, t)) {
-                        BOOST_LOG_TRIVIAL(warning) << "RE-allocation of " << t << " cell at col = " << col << ", row = " << row << ".\n";
+                     //   BOOST_LOG_TRIVIAL(warning) << "RE-allocation of " << t << " cell at col = " << col << ", row = " << row << ".\n";
```

Comment:
I don't want to remove it, because it is a good check for most use-cases where we don't use fresh_context (i.e. non-zkevm-opcodes circuits).

Alexey have a plan to adjust, so I leave it commented.